activesupport/lib/active_support/test_case.rb

```diff
@@ -55,6 +56,7 @@ def test_order
     include ActiveSupport::Testing::Assertions
     include ActiveSupport::Testing::Deprecation
     include ActiveSupport::Testing::TimeHelpers
+    include ActiveSupport::Testing::SilenceStream
```

Comment:
I don't think we want to make it publicly available for people to use outside of the framework, that's why it was deprecated and moved to the places it's currently in use. The best thing we can do is to find a way to share code between those places, without making it available through `AS::TestCase` I think.